component/vc/status/resolver/resolver.go

```diff
@@ -63,7 +76,67 @@ func (r *Resolver) Resolve(statusListVCURI string) (*verifiable.Credential, erro
 		return nil, fmt.Errorf("failed to parse and verify status vc: %w", err)
 	}
 
-	return vc, nil
+	return cred, nil
+}
+
+const (
+	idHubQueryMethod = "CollectionsQuery"
+)
+
+func (r *Resolver) resolveDIDRelativeURL(didURL string) ([]byte, error) {
+	docRes, err := r.didResolver.Resolve(strings.Split(didURL, "?")[0])
+	if err != nil {
+		return nil, fmt.Errorf("failed to resolve DID: %w", err)
+	}
+
+	didDoc := docRes.DIDDocument
+
+	queries, err := getQueries(didURL)
+	if err != nil {
+		return nil, err
+	}
+
+	objectID, reqMessage, err := identityhub.GetRequest(didDoc.ID, idHubQueryMethod, queries)
+	if err != nil {
+		return nil, fmt.Errorf("unable to construct identity hub request object: %w", err)
+	}
+
+	payload, err := json.Marshal(reqMessage)
+	if err != nil {
+		return nil, fmt.Errorf("unable to marshal identityHubRequest: %w", err)
+	}
+
+	svcEndpoint, err := identityhub.ServiceEndpoint(didDoc)
+	if err != nil {
+		return nil, fmt.Errorf("unable to find identity hub service endpoint in did doc: %w", err)
+	}
+
+	req, err := http.NewRequestWithContext(
```

Comment:
There is `client  *http.Client` in Resolver. But it not used here, is this intended?

Reply:
`Resolver.client` is used in `Resolver.sendHTTPRequest()`: https://github.com/hyperledger/aries-framework-go-ext/pull/335/files#diff-124ad46206dc099957fb9676ab0ed5b801023c2a8a7471564f1154eede2032ffR147.